cmd/algorand-indexer/daemon.go

```diff
@@ -175,7 +179,11 @@ func (bih *blockImporterHandler) HandleBlock(block *types.EncodedBlockCert) {
 	}
 	_, err := bih.imp.ImportDecodedBlock(block)
 	maybeFail(err, "ImportDecodedBlock %d: %v", block.Block.Round, err)
-	importer.UpdateAccounting(bih.db, bih.cache, uint64(block.Block.Round), genesisJSONPath, logger)
+	filter := idb.UpdateFilter{
+		StartRound: uint64(block.Block.Round)-1,
```

Comment:
This would break reset-rebuild.
Right now with UpdateAccounting doing its own fetch of accounting metastate it notices nothing is there and does the rebuild.

Reply:
do you think there's any value in doing a "catchup round accounting" elsewhere? I guess it would basically be this but run with:
```
    StartRound: db.GetMaxRoundAccounted() - 1,
    MaxRound: db.GetMaxRoundLoaded(),
```

Reply:
Yeah. It would be totally fair to explicitly do that before even trying to talk to algod. Expand the startup code to do more checks and things like that. Do the block-contiguiity check. Etc.

Reply:
I'm not going to take this on now, but I added it to the list in https://github.com/algorand/indexer/issues/279